### ISSUE TITLE
Un-Defunct Organizations

### ISSUE THREAD

**Issue by kicohen** [ABTech/abtt#331]
Would it be possible to have a way to mark defunct organizations as non-defunct.

**Comment by tedgarb** [comment]
Yes, but if you have an immediate need ping someone to edit the DB
